pkg/k8s/controllers/reconciler.go

```diff
@@ -0,0 +1,276 @@
+// Copyright 2021 Outreach Corporation. All Rights Reserved.
+
+// Description: This file defines a kubernetes controller for v1/PostgresqlDevenvDatabase.
```

Comment:
Shouldn't this ... not be in gobox?

Reply:
doh, nice catch -  I will remove copy-pasted code soon

Reply:
updated now, thanks